pkg/proxy/ipvs/proxier.go

```diff
@@ -1104,6 +1107,10 @@ func (proxier *Proxier) syncProxyRules() {
 				IP2:      epIP,
 				SetType:  utilipset.HashIPPortIP,
 			}
+			if valid, err := proxier.loopbackSet.validateEntry(entry); !valid {
+				glog.Errorf("Failed to add entry %v to set %s, error: %s, %v", entry, proxier.loopbackSet.Name, EntryInvalidErr, err)
```

Comment:
shouldnt we extract this string as a const?

Reply:
Because there are some variables in the string, e.g. $entry, $set, &err

What about:

```
const EntryInvalidErr = "error adding entry to ipset since entry is invalid"

glog.Errorf("%s, entry: %s, set: %s, reason: %s", EntryInvalidErr, entry, proxier.loopbackSet.Name, err)
```

Reply:
```
const EntryInvalidErr = "error adding entry to set since it's invalid"

glog.Errorf("%s", EntryInvalidErr)
```